tests/Issues/IndentationCrash/config/configured_rule.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Set\ValueObject\LevelSetList;
+use Rector\Set\ValueObject\SetList;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->sets([LevelSetList::UP_TO_PHP_81, SetList::CODE_QUALITY]);
```

Comment:
What are the minimals 2 rules causing the crash?
There will be some invalid node replacement. Probably missing set original node null.

Reply:
change `sizeof` to `count` seems on of it:

https://github.com/rectorphp/rector-src/blob/bb68d8aa18c2b0ab3f5000c175fd6b81e9c0610a/config/set/code-quality.php#L89

other seems need to check one by one set list from php 5.4 to 8.1

Reply:
applied rules reduced to RenameFucntionRector, ForRepeatedCountToOwnVariableRector, then LevelSetList::UP_TO_PHP_81 https://github.com/rectorphp/rector-src/pull/2760/commits/b05501d199883b3d46dc189f01190cb3597411ad

Reply:
Reduced to only 4 rules:

```
Rector\CodeQuality\Rector\For_\ForRepeatedCountToOwnVariableRector;
Rector\Config\RectorConfig;
Rector\Php71\Rector\FuncCall\CountOnNullRector;
Rector\Php81\Rector\FuncCall\NullToStrictStringFuncCallArgRector;
Rector\Renaming\Rector\FuncCall\RenameFunctionRector;
```

https://github.com/rectorphp/rector-src/pull/2760/commits/55efa059fc0a30377e0a9e40df943fee2e9acc4d

Reply:
https://getrector.org/demo/b3c0e6bc-073d-4cb1-a94e-37c2d06c64c1